### ISSUE TITLE
RuntimeError: generator raised StopIteration

### ISSUE THREAD

**Issue by nikitashalnov** [mkdocs/mkdocs#1531]
Hello

I faced with some problem with mkdocs. Yesterday all worked okay and my site, which hosts on Gitlab Pages, was successfully deployed. But today my gitlab ci jobs failed. Here is the more info.

I've crated a simple Dockerfile and tried to run `mkdocs build`.
Dockerfile:
```
FROM python:alpine
RUN pip install mkdocs
CMD ["/bin/sh"]
```
In container:
```
17:39 [master] n.shalnov:~/gitlab/pages/docs$ docker run --rm -it mkdocs sh
/ # mkdocs --version
mkdocs, version 0.17.4
/ # cd /opt/
/opt # mkdocs new my-project
INFO    -  Creating project directory: my-project 
INFO    -  Writing config file: my-project/mkdocs.yml 
INFO    -  Writing initial docs: my-project/docs/index.md 
/opt # cd my-project/
/opt/my-project # mkdocs build
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /opt/my-project/site 
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/mkdocs/nav.py", line 337, in _follow
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/mkdocs/__main__.py", line 156, in build_command
    ), dirty=not clean)
  File "/usr/local/lib/python3.7/site-packages/mkdocs/commands/build.py", line 282, in build
    build_pages(config, dirty=dirty)
  File "/usr/local/lib/python3.7/site-packages/mkdocs/commands/build.py", line 213, in build_pages
    site_navigation = nav.SiteNavigation(config)
  File "/usr/local/lib/python3.7/site-packages/mkdocs/nav.py", line 44, in __init__
    config, self.url_context)
  File "/usr/local/lib/python3.7/site-packages/mkdocs/nav.py", line 392, in _generate_site_navigation
    config_line, url_context, config):
RuntimeError: generator raised StopIteration
```

So my Gitlab Job (without any changes from yesterday) fails with the same error.

What can be the cause of the problem?

**Comment by waylan** [comment]
You appear to be running on Python 3.7, which the current version of MkDocs does not support. This will be fixed in the next release. In the meantime, you will need to use Python 3.6 or older. I'm closing this as a duplicate of #1518.

**Comment by waylan** [comment]
Version 0.17.5 is now available with the fix in #1536.
